apps/nextjs-app-router/src/app/layout.tsx

```diff
@@ -1,17 +1,18 @@
 import { MakeswiftProvider } from '@/makeswift/provider'
 import { DraftModeScript } from '@makeswift/runtime/next/server'
 
-type Params = { lang: string; path?: string[] }
+type Params = Promise<{ lang: string; path?: string[] }>
 
-export default function RootLayout({
+export default async function RootLayout({
```

Comment:
Breaking Next.js 15 change: `RootLayout` become async to access async `params`; we could keep it synchronous and wrap `params` with `React.use` (https://nextjs.org/docs/app/building-your-application/upgrading/version-15#params--searchparams), but that's also a breaking change, so I opted for async as more forward-looking / consistent with the `Page` component.